geoengine/datasets.py

```diff
@@ -425,13 +425,15 @@ def pandas_dtype_to_column_type(dtype: np.dtype) -> FeatureDataType:
 
 def upload_dataframe(
         df: gpd.GeoDataFrame,
-        name: str = "Upload from Python",
+        display_name: str = "Upload from Python",
+        name: Optional[str] = None,
```

Comment:
Don't you have to update at least one example?

Reply:
I think the upload example does not use the "name". I can add one if you want.